lib/sober_swag/output_object/field_syntax.rb

```diff
@@ -37,10 +37,11 @@ def primitive(name)
       # Note that merging in a full output object *will not* also merge in views, just fields defined on the base.
       #
       # @param other [#fields] a field container, like a {SoberSwag::OutputObject} or something
+      # @param opts [Hash] accepts a key of :except to optionally exclude a field from the output object being merged
       # @return [void]
-      def merge(other)
+      def merge(other, opts = {})
```

Comment:
```suggestion
      def merge(other, except: [])
```

Can we use this? I prefer keyword arguments over an opts hash.

Reply:
+1